src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -178,44 +184,9 @@ const IssueForm = (): JSX.Element | null => {
   }
 
   if (status === STATUSES.RESOLVED) {
-    const fetchVaults = async (): Promise<Map<AccountId, BitcoinAmount> | undefined> => {
-      try {
-        // This data (the vaults) is strictly required to request issue
-        const vaults = await window.bridge.vaults.getVaultsWithIssuableTokens();
```

Comment:
This call is happening on submit for a reason - see:
https://github.com/interlay/interbtc-ui/pull/180